### PR TITLE
Document using openshift-cherrypick-robot

### PR DESCRIPTION
**Pull request by fultonj**

### CODE REVIEW

File: docs/contributing/cherry-picking.md [hunk 1]
```diff
@@ -14,7 +14,24 @@ cherry pick.
 
 Please do not ask [core maintainers](../../OWNERS) to cherry pick your
 patch though we will be happy to review it and help merge it once the
-cherry pick PR has been submitted.
+cherry pick PR has been submitted using one of the methods below.
+
+## Method 1
+
+After your PR merges, add a comment like the following to it.
```
Comment (abays) [inline docs/contributing/cherry-picking.md:21]: You can also add the comment before the PR merges, and the bot will only create the new PR after the current PR merges

Comment (fultonj) [reply docs/contributing/cherry-picking.md]: OK, my latest update contains this information.

**Review by abays — APPROVED** [review]
/lgtm

**Review by abays — APPROVED** [review]
/approve

**Pull request merged**